code/go/0chain.net/blobbercore/challenge/challenge.go

```diff
@@ -135,49 +137,112 @@ func processAccepted(ctx context.Context) {
 	}()
 
 	db := datastore.GetStore().GetDB()
-	challenges := make([]*ChallengeEntity, 0)
-	db.Where(ChallengeEntity{Status: Accepted}).Find(&challenges)
-	if len(challenges) > 0 {
-
-		startTime := time.Now()
-
-		swg := sizedwaitgroup.New(config.Configuration.ChallengeResolveNumWorkers)
-		for _, c := range challenges {
-			logging.Logger.Info("[challenge]process: ",
-				zap.String("challenge_id", c.ChallengeID),
-				zap.Time("created", c.CreatedAt))
-			err := c.UnmarshalFields()
-			if err != nil {
-				logging.Logger.Error("[challenge]process: ",
-					zap.String("challenge_id", c.ChallengeID),
-					zap.Time("created", c.CreatedAt),
-					zap.String("validators", string(c.ValidatorsString)),
-					zap.String("lastCommitTxnList", string(c.LastCommitTxnList)),
-					zap.String("validationTickets", string(c.ValidationTicketsString)),
-					zap.String("ObjectPath", string(c.ObjectPathString)),
-					zap.Error(err))
-				continue
-			}
-			swg.Add()
-			go validateChallenge(&swg, c)
+
+	rows, err := db.Model(&ChallengeEntity{}).
+		Where("status = ?", Accepted).
+		Select("challenge_id", "created_at").Rows()
+
+	if err != nil {
+		logging.Logger.Error("[challenge]process: ",
+			zap.Error(err))
+		return
+	}
+
+	defer rows.Close()
+
+	startTime := time.Now()
+	swg := sizedwaitgroup.New(config.Configuration.ChallengeResolveNumWorkers)
+	count := 0
+	for rows.Next() {
+		count++
+		now := time.Now()
+
+		var challengeID string
+		var createdAt time.Time
+
+		err := rows.Scan(&challengeID, &createdAt)
+		if err != nil {
+			logging.Logger.Error("[challenge]process: ",
+				zap.Error(err))
+			continue
 		}
-		swg.Wait()
 
-		logging.Logger.Info("[challenge]elapsed:process ",
-			zap.Int("count", len(challenges)),
-			zap.String("save", time.Since(startTime).String()))
+		if time.Since(createdAt) > config.Configuration.ChallengeCompletionTime {
```

Comment:
I think `created_at` timestamp is taken from time challenge was created from /openchallenges API
If blobbers server have different timezone, time difference this is an issue.

Reply:
it should be ok. because @guruhubb asked devops to set all server with UTC.

Reply:
see https://github.com/0chain/blobber/issues/219

Reply:
All servers should sync the time when they are setup, perhaps `ntp` is required, cc @guruhubb